Firebase/InstanceID/FIRInstanceIDTokenManager.m

```diff
@@ -281,12 +281,12 @@ - (BOOL)checkTokenRefreshPolicyWithIID:(NSString *)IID {
   NSMutableArray<FIRInstanceIDTokenInfo *> *tokenInfosToDelete =
       [NSMutableArray arrayWithCapacity:tokenInfos.count];
   for (FIRInstanceIDTokenInfo *tokenInfo in tokenInfos) {
-    BOOL isTokenFresh = [tokenInfo isFresh];
-    if (isTokenFresh && [tokenInfo.token hasPrefix:IID]) {
+    BOOL isTokenFresh = [tokenInfo isFreshWithIID:IID];
+    if (isTokenFresh) {
```

Comment:
nit: Doesn't look like the variable is used anywhere else. If so, can we just call the method directly in `if`?